account_vat_period_end_statement/README.rst

```diff
@@ -111,6 +111,7 @@ Contributors
 * Alex Comba <alex.comba@agilebg.com>
 * Alessandro Camilli <camillialex@gmail.com>
 * Simone Rubino <simone.rubino@agilebg.com>
+* Giacomo Grasso <giacomo.grasso.82@gmail.com>
```

Comment:
You can autogenerate a nice README using [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)